### PR TITLE
June 29 17 fixes

### PR DESCRIPTION
**Pull request by rosepierce**

### CODE REVIEW

File: recaptcha/CTL000179.yaml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 bioguide: CTL000179
 contact_form:
   method: post
-  action: 
+  action:
   steps:
-    - visit: "http://cthousegop.com/contact-gail/"
+    - visit: "http://www.cthousegop.com/lavielle/contact-me/"
```
Comment (tomdionesotes) [inline recaptcha/CTL000179.yaml]: In the future, if a target has recaptcha V2, you don't need to fix up the yaml - just move it to the recaptcha folder. It's fine to leave this though.

**Comment by tomdionesotes** [comment]
Could you also move E000295 to the recaptcha folder?

**Comment by rosepierce** [comment]
yes, I just pushed the changes I made yesterday here and have another branch open that I moved E000295 in and have another form with changes in. should I include those in this instead?

**Comment by tomdionesotes** [comment]
Nah, today's changes can be a separate PR. This looks good, go ahead and merge.

**Pull request merged**